test/e2e/framework.go

```diff
@@ -146,7 +146,11 @@ func (f *Framework) afterEach() {
 					timeout = f.NamespaceDeletionTimeout
 				}
 				if err := deleteNS(f.Client, ns.Name, timeout); err != nil {
-					Failf("Couldn't delete ns %q: %s", ns.Name, err)
+					if !strings.Contains(err.Error(), "not found") {
```

Comment:
Use [IsNotFound](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/errors/errors.go#L364) so you don't have to do string matching.